bin/tests/install.bats

```diff
@@ -44,10 +44,10 @@ load 'common.sh'
   answer="jjj" #jjj as 3 times the down arrow keys to select the html output because it doesn't ask any other question
   run restqa i -c ./bin/tests/features/success/.restqa.yml -e prod <<< $answer
   assert_failure
-  assert_output --partial ">  Error: \"prod\" is not an environment available in the config file, choose between : local"
+  assert_output --partial ">  Error: \"prod\" is not an environment available in the configuration file, choose between : local"
```

Comment:
@KaTLaz this line should not change because its breaking the test.

Reply:
reverted it